templates/properties/ldap-healthcheck-config.json.erb

```diff
@@ -0,0 +1,47 @@
+{
+   "ldap-config" : {
+<% if defined? @ldap_server -%>
+     "server" : "<%=@ldap_server%>",
+<% end -%>
+<% if ldap_port -%>
+     "port" : <%=ldap_port%>,
+<% end -%>
+<% if defined? @ldap_serach_base -%>
+     "serach_base" : "<%=@ldap_serach_base%>",
+<% end -%>
+<% if defined? @ldap_username -%>
+     "username" : "<%=@ldap_username%>",
+<% end -%>
+<% if defined? @ldap_password -%>
+     "password" : "<%=@ldap_password%>",
+<% end -%>
+<% if ldap_pool_size -%>
+     "pool_size" : <%=ldap_pool_size%>,
+<% end -%>
+<% if ldap_use_tls -%>
+     "use_tls" : <%=ldap_use_tls%>,
+<% end -%>
+<% if defined? @truststore_url -%>
+     "truststore" : "<%=@truststore_url%>",
+<% end -%>
+<% if defined? @truststore_password -%>
+     "truststore_password" : "<%=@truststore_password%>",
+<% end -%>
+<% if defined? @connectionTimeoutMS -%>
+     "connectionTimeoutMS" : "<%=@connectionTimeoutMS%>",
+<% end -%>
+<% if defined? @responseTimeoutMS -%>
+     "responseTimeoutMS" : "<%=@responseTimeoutMS%>",
+<% end -%>
+<% if debug -%>
+     "debug" : <%=debug%>,
+<% end -%>
+<% if keepAlive -%>
+     "keepAlive" : <%=keepAlive%>,
+<% end -%>
+<% if defined? @poolMaxConnectionAgeMS -%>
+     "poolMaxConnectionAgeMS" : "<%=@poolMaxConnectionAgeMS%>",
+<% end -%>
+     "documentation" : "This is LDAP configuration to initialize RolesProvider."
+  }
+}
```

Comment:
Do we really need all the above parameters to create Ldap configuration as a lot of these fields already have a default value and since this connection will be used only for healthcheck, defaults should suffice.